AlfrescoApp/Utils/Subclasses/ALFSwipeToDeleteGestureRecognizer.h

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of the Alfresco Mobile iOS App.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#import <UIKit/UIKit.h>
+
+@interface ALFSwipeToDeleteGestureRecognizer : UIPanGestureRecognizer
+
+- (void) endGestureHandling;
```

Comment:
It's not critical (and I know we've not followed this before) but it's good practice to prefix the category's method name so as not to clash with any future Apple updates to the API. i.e call this `alf_endGestureHandling`